custom_components/kia_uvo/Vehicle.py

```diff
@@ -83,18 +84,20 @@ def login(self):
         self.token = self.kia_uvo_api.login()
 
     def set_last_updated(self):
+        kia_tz = pytz.timezone("Europe/Paris")
```

Comment:
can we use constants here?

Reply:
Updated to use constants. Also now normalizing to use UTC for both side of the subtraction, so it isn't affected by daylight savings offsets changing for timezone time, but not UTC.